### PR TITLE
 feat(docker compose): dev and prod

### LINKED ISSUE #65: Docker compose: dev and prod
Create a second docker compose to use it for prod

This docker compose will build each app and run them without watch. It will be used for deployement: Each dockerfile will create an image with will be used in docker compose without using hot reloading

### PR DESCRIPTION
**Pull request by Miou-zora**
Linked to:
- close #65

### CODE REVIEW

**Comment by Miou-zora** [comment]
I've maybe fixed dependabot warnings

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      - common-volume:/common-volume
+
+  client_web:
+    restart: always
+    container_name: client_web
+    image: client_web_dev
+    build:
+      context: ./web
+      dockerfile: Dockerfile.dev
+      # args:
```
Comment (Benjicatch) [inline docker-compose.dev.yml:71]: to remove

**Review by Benjicatch — CHANGES_REQUESTED** [review]

File: docker-compose.dev.yml (hunk 1 above)
Comment (Miou-zora) [reply docker-compose.dev.yml]: done

**Review by Benjicatch — APPROVED** [review]
ok

**Review by Pablodeibar — APPROVED** [review]
looks good

**Review by Benjicatch — APPROVED** [review]

**Review by Pablodeibar — APPROVED** [review]
good me too

**Pull request merged**